pkg/parser/external/java_parser_server.go

```diff
@@ -37,8 +38,13 @@ func getServerPort() string {
 }
 
 func isServerUp() bool {
-	cmd := exec.Command("curl", "-v", fmt.Sprintf("localhost:%s", getServerPort()))
-	if err := cmd.Run(); err != nil {
+	conn, err := net.Dial("tcp", fmt.Sprintf(":%s", getServerPort()))
```

Comment:
Shall we set a `timeout` value? How about add a unit test for this function.

Reply:
Good point. I will add a unit test.